index.js

```diff
@@ -92,7 +106,10 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  if (movies.length === 0) {return []}
```

Comment:
I know you know how to throw errors, since you did it in other ones.

If you weren't sure why your tetss were failing, practic reading the test results in the termianl and the test file itself. The tests are telling you you're not throwing, so a quick change in many of these functions would have gotten you a lot more points!

If you simply ran out of time, then practice until you're able to solve these problems more quickly. It makes perfect!